decisions/wc_class_decisions.txt

```diff
@@ -3340,13 +3340,15 @@ targetted_decisions = {
 
 				increase_visibility_effect = yes
 				minor_soul_corruption_effect = yes
-				if = {
-					limit = {
-						has_quest = quest_turn_into_undead
-						quest_target = { character = ROOT }
+				tooltip = {
+					if = {
+						limit = {
+							has_quest = quest_turn_into_undead
+							quest_target = { character = ROOT }
+						}
+						clr_quest = quest_turn_into_undead
+						add_society_currency_medium_effect = yes
 					}
```

Comment:
This tooltip is misleading, since you actually don't complete this quest until you choose the right option in the following event.

Reply:
**SAD**